src/features/jumpNav.tsx

```diff
@@ -188,17 +189,38 @@ const toggleLinkedRefs = () => {
   ).click();
   document.querySelector(".rm-reference-container .rm-caret").scrollIntoView();
 };
-const toggleUnlinkedRefs = () => {
-  (
-    document.querySelector(
-      ".rm-reference-main > div > div:nth-child(2) > div > span > span"
-    ) as HTMLElement
-  ).click();
-  document
-    .querySelector(
-      ".rm-reference-main > div > div:nth-child(2) > div > span > span"
-    )
-    .scrollIntoView();
+const toggleUnlinkedRefs = async () => {
+  const rmReferenceDiv = document.querySelector(
+    ".rm-reference-main"
+  ) as HTMLElement;
+  const textEl = Array.from(
+    rmReferenceDiv.querySelectorAll(":scope *:not(:empty):not(:has(*))")
+  ).find((element) => element.textContent?.includes("Unlinked References"));
+  const unlinkedRefsCaret = textEl
+    ?.closest("div")
+    ?.querySelector("span.rm-caret") as HTMLElement;
+
+  if (!unlinkedRefsCaret) {
+    const uid = await window.roamAlphaAPI.ui.mainWindow.getOpenPageOrBlockUid();
+    if (!uid) {
+      renderToast({
+        id: "dnp-refs-toast",
+        content: "No Unlinked References on Daily Notes Page",
+        intent: "warning",
+      });
+    } else if (uid && getParentUidsOfBlockUid(uid).length > 0) {
+      renderToast({
+        id: "block-refs-toast",
+        content: "No Unlinked References on a block",
+        intent: "warning",
+      });
+    } else {
+      throw new Error();
```

Comment:
I'm curious what a good way to do this would look like.  Would we extend `Error`?

I initially tried to pass a message to `Error`, which worked fine, but it seemed I had to include `e.message` in the catch of `addCommand`.  

This meant if an error didn't pass a custom message, the generic message would be displayed on the toast.  I assumed we were trying to avoid that from [this comment](https://github.com/dvargas92495/roamjs-workbench/pull/414#discussion_r1230282065) (But I may have conflated "prompt to check the console" with "show the console message")

Reply:
yea I was mostly avoiding prompting the user to check the console - in general, we want to pass in some form of custom message that either we read (via the email) or the user reads (vai the toast)